frontend/src/modules/Shared/Feed/FeedCommentAdd.js

```diff
@@ -0,0 +1,99 @@
+import SendIcon from '@mui/icons-material/Send';
```

Comment:
The PR should be rebased on top of the fresh main. `modularization/frontend` has `FeedCommentAdd.js` under different directory

Reply:
PR updated with https://github.com/awslabs/aws-dataall/pull/600/commits/38c2e340ee3ac577c5214267aaabfc7a3f302f00